### PR TITLE
Gpuciscripts clean and update

### PR DESCRIPTION
**Pull request by msadang**
In CI folder the following changes have been made:

- Remove logger function and replace all logger calls with gpuci_logger

- Removed all ellipses ... from logger messages

- Prepend /opt to the conda path in PATH variable

- Replace conda with gpuci_conda_retry for build and install calls  (Did not replace conda activate with gpuci_conda_retry) 

- Replace source activate <env> with the following:
. /opt/conda/etc/profile.d/conda.sh
conda activate rapids

- Replace conda list with more verbose information:
conda info
conda config --show-sources
conda list --show-channel-urls

- Update Copyright year in the top of scripts to include 2020 if applicable

- Set PARALLEL_LEVEL to ${PARALLEL_LEVEL:-4}

- Set gpuci_conda_retry flags in cpu/build.sh
Setup 'gpuci_conda_retry' for build retries (results in 2 total attempts)
export GPUCI_CONDA_RETRY_MAX=1
export GPUCI_CONDA_RETRY_SLEEP=30

- Replace calls to gcc and g++ with $CC and $CXX respectively

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Review by ajschmidt8 — CHANGES_REQUESTED** [review]
looks good so far. I found a few things that will need to be changed. Dillon's `rmm` PR below is a good reference (just ignore the project flash specific stuff).

https://github.com/rapidsai/rmm/pull/615/files

File: ci/local/README.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ For a full list of available gpuCI docker images, visit our [DockerHub](https://
 Style Check:
 ```bash
 $ bash ci/local/build.sh -r ~/rapids/cuml -s
-$ source activate gdf    #Activate gpuCI conda environment
+$ . /opt/conda/etc/profile.d/conda.sh
+$ conda activate rapids gdf    #Activate gpuCI conda environment
```
Comment (dillon-cullinan) [inline ci/local/README.md:34]: `gdf` should be removed here.

**Review by dillon-cullinan — CHANGES_REQUESTED** [review]

File: ci/local/README.md (hunk 1 above)
Comment (msadang) [reply ci/local/README.md]: gdf removed

**Comment by msadang** [comment]
gpuci_logger and parallel level changed.

**Review by ajschmidt8 — APPROVED** [review]

**Review by dillon-cullinan — APPROVED** [review]

**Pull request merged**